mongo/database.go

```diff
@@ -523,6 +562,63 @@ func (db *Database) Watch(ctx context.Context, pipeline interface{},
 	return newChangeStream(ctx, csConfig, pipeline, opts...)
 }
 
+// mergeCreateCollectionOptions combines the given CreateCollectionOptions instances into a single
+// CreateCollectionOptions in a last-one-wins fashion.
```

Comment:
Optional: see similar for explanation

```suggestion
// CreateCollectionOptions in a last-property-wins fashion.
```